### ISSUE TITLE
Wrong mount defaults on FreeBSD (/etc/fstab: Inappropriate file or format)

### ISSUE THREAD

**Issue by sekrett** [chef/chef#4959]
## Description

/etc/fstab must de written in a different way on FreeBSD
## Chef Version

12.10.24
## Platform Version

bento/FreeBSD-10.3
## Replication Case

```
directory '/source'
directory '/destination'
mount '/destination' do
  device '/source'
  fstype 'nullfs'
  pass 0
  action [:mount, :enable]
end
```
## Client Output

It adds:
`/source /destination nullfs defaults 0 0`

This does not mount after reboot. It should be:
`/source /destination nullfs rw 0 0`

So `defaults` does not work on FreeBSD and must be replaced.
## Workaround

Add `options 'rw'`.

**Comment by cramaechi** [comment]
I think the substitution should take place in `.../lib/chef/provider/mount/mount.rb`:

<img width="1204" alt="screen shot 2018-01-21 at 3 31 02 am" src="https://user-images.githubusercontent.com/24444174/35192769-c091fc02-fe5d-11e7-8a5f-7abd67f8c312.png">

My only problem now is figuring out whether this change affects all platforms.

**Comment by cheeseplus** [comment]
Fixed by #6782
